### PR TITLE
Return steward name and email on GET

### PR DESCRIPTION
**Pull request by cecilia-donnelly**
Also update documentation. The other endpoints were correctly returning the email instead of the account id, but this one was not. @luciancerbu-vsp also requested that we return the name, so I included that - let me know if that'll be a problem.

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -129,7 +129,8 @@ instead return a [Joi ValidationError object](https://joi.dev/api/?v=17.9.1#vali
       createdDt: string (date),
       updatedDt: string (date),
     },
-    stewardAccountId: string,
+    stewardEmail: string,
+    stewardName: string,
```
Comment (liam-lloyd) [inline API.md:133]: If we're returning multiple properties of the steward, I think we should make the steward its own object within our directive object.
```
steward: {
  email: string,
  name: string,
}
```

Comment (liam-lloyd) [reply API.md:133]: Of course that probably breaks `web-app`'s integration with this endpoint, which isn't ideal.

Comment (cecilia-donnelly) [reply API.md:133]: That makes sense - @meisekimiu let's discuss in planning.

**Comment by cecilia-donnelly** [comment]
Updated to return a steward object on GET. I didn't change the other endpoints to include the name, though maybe I should. What do you think, @luciancerbu-vsp and @meisekimiu ?

**Comment by luciancerbu-vsp** [comment]
> Updated to return a steward object on GET. I didn't change the other endpoints to include the name, though maybe I should. What do you think, @luciancerbu-vsp and @meisekimiu ?

I think it will be nice for the other endpoints also to include the name.

**Comment by cecilia-donnelly** [comment]
Updated so creating, updating, and getting a directive all return a steward object with name and email.

**Review by liam-lloyd — COMMENTED** [review]
Everything looks good, but we should update the `Directive` interface in `src/directive/model.ts` to reflect this change - unfortunately because Typescript doesn't enforce itself at runtime nothing actually breaks if the type we say we're returning from these endpoints isn't what we actually get back from the database.

**Comment by cecilia-donnelly** [comment]
Added that type change - thanks for pointing it out! Didn't even think of it.

File: src/directive/model.ts [hunk 2]
```diff
@@ -35,6 +35,7 @@ export interface Directive {
   updatedDt: Date;
   trigger: DirectiveTrigger;
   stewardAccountId?: string;
```
Comment (liam-lloyd) [inline src/directive/model.ts:37]: Commented on the commit already (unintentionally, I didn't know you could do that until just now), but I think we should remove the `stewardAccountId` if it's only used in tests; tests shouldn't be able to influence what our models look like.

Comment (cecilia-donnelly) [reply src/directive/model.ts]: Done.

**Review by liam-lloyd — APPROVED** [review]

**Pull request merged**